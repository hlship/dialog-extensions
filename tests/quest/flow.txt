n
ask why
z
no
yes
