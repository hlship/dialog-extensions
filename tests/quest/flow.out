

Long have you searched for a quest worthy for you to submit yourself before the
kind. Perhaps this is the day?

Verdant Valley
A verdant valley; northwards lies a menacing pass through the mountains.

> n
You walk north.

Menacing Pass
A narrow pass through the jagged mountains leading from the valley (to the
south) northwards to unknown territory.

A wizened old hermit blocks your path, muttering something about quests and
dragons under his breath.

The hermit runs up to you, blocking your path forward.

You could ask why he is blocking the way.

> ask why
"Why dost thou block my way, hermit?" you ask.

"A dragon has awoke north of here and some brave soul must undertake the task to
defeat it before it lays waste to the valley yonder" he replies.

> z
A moment slips away.

"Wilt thou accept this quest, hero?"

"Though must!" insists the hermit.

You could say yes or say no.

> no
"No."

"But thou must!"

You could say yes.

> yes
"Yes."

"Truly, thou art brave!".

> 
